src/conspiracies/prompt_relation_extraction/data_classes.py

```diff
@@ -0,0 +1,538 @@
+"""Pydantic data classes for the relation extraction using prompt-based
+models."""
+
+from copy import copy
+from functools import partial
+from typing import Any, Dict, Iterable, List, Literal, Optional, Tuple, Union
+
+import spacy
+from pydantic import BaseModel
+from spacy import displacy
+from spacy.tokens import Doc, Span, Token
+
+
+def get_text(span: Iterable[Token], ignore_spaces: bool, lowercase: bool) -> str:
+    """Gets the text of a span or doc.
+
+    Args:
+        span (Union[Span, Doc]): a spacy span or doc.
```

Comment:
Type hint does not match function definition type hint

Reply:
(And other input arguments are not documentet)

Reply:
Updated to match type hint